src/SettingsView/EntityMappingListItem.js

```diff
@@ -64,7 +64,9 @@ class EntityMappingListItem extends ConnectionComponent {
 						<IconButton><IconEdit/></IconButton>
 						<ConfirmationMessage
 							actionToPerform={this.deleteMapping}
-							ref={(input) => {this.confirmationMessage = input;}}>
+							ref={(input) => {this.confirmationMessage = input;}}
+							onSnackbarMessage={this.props.onSnackbarMessage}

```

Comment:
Where does the onSnackbarMessage property come from? I don't understand where it comes from in the other components either.